embedding/embedlite/embedthread/EmbedLiteCompositorParent.cpp

```diff
@@ -237,10 +239,33 @@ void EmbedLiteCompositorParent::SetSurfaceSize(int width, int height)
   SetEGLSurfaceSize(width, height);
 }
 
-void EmbedLiteCompositorParent::SetScreenRotation(const mozilla::ScreenRotation &rotation, const gfx::Matrix &matrix)
+void EmbedLiteCompositorParent::SetScreenRotation(const mozilla::ScreenRotation &rotation)
 {
   if (mRotation != rotation) {
-    mWorldTransform = matrix;
+    gfxMatrix rotationMartix;
```

Comment:
Since the API doesn't depend on gfxMatrix it makes sense to use gfx::Matrix instead here. And gfx::Point instead of gfxPoint. The latter is going to be deprecated in future versions.